js_modules/dagit/packages/core/src/runs/RunTimeline.tsx

```diff
@@ -56,46 +68,120 @@ export const RunTimeline = (props: Props) => {
   }, []);
 
   const [unvalidatedStart, unvalidatedEnd] = range;
-
   // Ensure that `start` is earlier than `end`.
   const [start, end] = React.useMemo(() => {
     return unvalidatedEnd < unvalidatedStart
       ? [unvalidatedEnd, unvalidatedStart]
       : [unvalidatedStart, unvalidatedEnd];
   }, [unvalidatedStart, unvalidatedEnd]);
 
-  const jobList = React.useMemo(() => {
-    const jobKeys = Object.keys(jobs);
-    const overlapComparison = {start, end};
+  const queryResult = useQuery<RunTimelineQuery, RunTimelineQueryVariables>(RUN_TIMELINE_QUERY, {
```

Comment:
It would be nice to keep `RunTimeline` a non-GraphQL component, since it's helpful to have storybook examples with `faker` data. I also haven't written any tests for it yet, but when I do, it will be easier to do so with `faker` data than by mocking GraphQL.

Can you instead wrap `RunTimeline` with a GraphQL-ful component and pass `RunTimeline` the result?

Reply:
Yeah, I'll pull this up.